UPDATING.md

```diff
@@ -219,6 +219,19 @@ if foo is None:
 
 This changes the behaviour if you previously explicitly provided `None` as a default value. If your code expects a `KeyError` to be thrown, then don't pass the `default_var` argument. 
 
+### Removal of `airflow_home` config setting
+
+There were previously two ways of specifying the Airflow "home" directory
+(`~/airflow` by default): the `AIRFLOW_HOME` environment variable, and the
+`airflow_home` config setting in the `[core]` section.
+
+If they had two different values different parts of the code base would end up
+with different values. The config setting has been deprecated, and you should
+remove the value form the config files and set `AIRFLOW_HOME` environment
+variable if you need to use a non default value for this.
+
+(Since this setting is used to calculate what config file to load, it is not
+possible to keep just the config option)
```

Comment:
@ashb I'm wondering what will happen if I have both env variables `AIRFLOW_HOME` and `AIRFLOW__CORE__AIRFLOW_HOME` , but set to different values?

Reply:
To the code the second one is the same as setting it in the config file

Reply:
Having re-read this: this is the very problem I ran in to that caused me to issue the PR. If the are set to different values then the webserver_config.py would be looked for in one, but other things from the other.

Specifically the problem I had was that webserver_config.py was written to one, but read from the other! So `airflow create_user` etc targeted an `sqlite://:memory:` DB!